indra/newview/scripts/lua/LLDebugSettings.lua

```diff
@@ -0,0 +1,17 @@
+leap = require 'leap'
+
+local LLDebugSettings = {}
+
+function LLDebugSettings.set(name, value)
+    return leap.request('LLViewerControl', {op='set', group='Global', key=name, value=value})
```

Comment:
Should `leap.request()` and `leap.generate()` notice an `"error"` key in the returned data and raise the specified error?

If they don't, maybe we should do it here and in `toggle()`.